### PR TITLE
adjust text warm up

### PR DESCRIPTION
**Pull request by Rensvandeschoot**
This PR proposes to adjust the text in the warm up screen.
localhost_5000_projects

### CODE REVIEW

File: asreview/webapp/src/ProjectComponents/SetupComponents/FinishSetup.js [hunk 1]
```diff
@@ -112,8 +112,18 @@ const FinishSetup = (props) => {
...
               </TypographySubtitle1Medium>
+              <Typography
+                 variant="body2"
+                 sx={{ color: "text.secondary" }}
+               >
+                      While we warm up the AI, you can learn more about 
+                      the workings of ASReview LAB by watching the video. 
+                      Meanwhile, we initialize the project file, 
+                      extract the features, and run the classifier 
+                      with the selected prior knowledge.
```
Comment (J535D165) [inline asreview/webapp/src/ProjectComponents/SetupComponents/FinishSetup.js:125]:
```suggestion
                      While ASReview LAB warms up the AI, you can learn more about 
                      its works by watching the video. 
                      Meanwhile, your features are extracted, and the classifier 
                      is trained with the selected prior knowledge.
```

File: asreview/webapp/src/ProjectComponents/SetupComponents/FinishSetup.js [hunk 2]
```diff
@@ -128,7 +138,7 @@ const FinishSetup = (props) => {
               <Fade in>
                 <Stack spacing={3} className={classes.root}>
                   <TypographySubtitle1Medium>
-                    Your project is ready
+                    Your project has been initiated
```
Comment (J535D165) [inline asreview/webapp/src/ProjectComponents/SetupComponents/FinishSetup.js:141]:
```suggestion
                    You can start screening!
```

File: asreview/webapp/src/ProjectComponents/SetupComponents/FinishSetup.js (hunk 1 above)
Comment (J535D165) [reply asreview/webapp/src/ProjectComponents/SetupComponents/FinishSetup.js:125]: Unclear who "we" is, so rewrote it a bit. Text seems good to me :)

**Review by terrymyc — COMMENTED** [review]
Thanks. I rephrased and formatted it a bit.
image
image

**Pull request merged**